<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddS</class>
 <widget class="QDialog" name="DialogAddS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>添加</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_ObjectName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名次/排位：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_Range"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>奖项名称：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>级别类型：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_LevelClass"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>奖项类型：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_AwardClass">
       <item>
        <property name="text">
         <string>请选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>院级以上竞赛</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>科技学术论文</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbt_Cancel">
       <property name="text">
        <string>完成</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/Done.ico</normaloff>:/images/Done.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_Ok">
       <property name="text">
        <string>添加/修改</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/add.ico</normaloff>:/images/add.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>已有项</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbt_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
